src/npc_class.cpp

```diff
@@ -267,7 +267,7 @@ distribution distribution::dice_roll( int sides, int size )
     } );
 }
 
-distribution distribution::operator+( const distribution &other ) const
+distribution distribution::sum( const distribution &other ) const
```

Comment:
Perhaps the problem here is non-idiomatic operators? That scent one is pretty ugly because it doesn't do anything like you would expect?